src/js/server/models/Hacker.js

```diff
@@ -184,15 +193,26 @@ Hacker.upsertAndFetchFromMlhUser = function (mlhUser) {
 };
 
 // Determine the headline application status
-Hacker.deriveOverallStatus = function (applicationStatus, responseStatus, teamApplicationStatus, furtherDetailsStatus) {
+Hacker.deriveOverallStatus = function (applicationStatus, responseStatus, teamApplicationStatus, rsvpStatus) {
   if (applicationStatus == statuses.application.INCOMPLETE || teamApplicationStatus == statuses.application.INCOMPLETE)
     return statuses.overall.INCOMPLETE;
   else if (responseStatus == statuses.response.PENDING)
     return statuses.overall.IN_REVIEW;
   else if (responseStatus == statuses.response.REJECTED)
     return statuses.overall.REJECTED;
-  else if (furtherDetailsStatus == statuses.furtherDetails.INCOMPLETE)
-    return statuses.overall.ACCEPTED_INCOMPLETE;
-  else if (furtherDetailsStatus == statuses.furtherDetails.COMPLETE)
-    return statuses.overall.ACCEPTED_COMPLETE;
+  else if (rsvpStatus == statuses.rsvp.INCOMPLETE)
+    return statuses.overall.INVITED_AWAITING_RSVP;
+  else if (rsvpStatus == statuses.rsvp.COMPLETE_NO)
+    return statuses.overall.INVITED_DECLINED;
+  else if (rsvpStatus == statuses.rsvp.COMPLETE_YES)
+    return statuses.overall.INVITED_COMPLETE;
```

Comment:
I think semantically this makes sense as the user has been `INVITED` and has `COMPLETE` their RSVP form. We could add another state `HAS_TICKET` or something like that to distinguish between those but in our setup, people who are invited and have RSVP'd _should_ have tickets so it's reasonable to work with the assumption that they do and throw an error if it turns out that they don't as this indicates something has gone wrong. What do you reckon?